docs/versionIndex.md

```diff
@@ -46,6 +46,15 @@ backend services have been tested with each released version of ADF.
 -   [v2.1.0](#v210)
 -   [v2.0.0](#v200)
 
+## v6.10.0
```

Comment:
The entry for 6.10.0 is missing in the [main index](https://github.com/Alfresco/alfresco-ng2-components/pull/9801/files#diff-6825824005472290de3d91fdefb2061938ad16c97c550d35b5fba0ea47a7eb00R13) near the top of the file

Reply:
right, added, ty